### ISSUE TITLE
Probably worth changing repo name :)

### ISSUE THREAD

**Issue by RedundantCich** [lioneltrs/goNCommand#8]
It has grown a lot since you started with Ninja Commando :D 
Worth changing the name to something more catchy :)

**Comment by lioneltrs** [comment]
You are right! I had no idea it would go beyond Ninja Commando! Do you have any idea for an appropriate name? convertROMS?

**Comment by RedundantCich** [comment]
The naming is always the most difficult in programming :D 

convertROMS sounds cool :)

**Comment by Lx32** [comment]
I think that we need to put in the name that the rom are from Amazon Prime, in this way will be more searchable for other users.

**Comment by lioneltrs** [comment]
There is a risk because Prime Gaming or Amazon Prime are trademarks, as far as I understand.

**Comment by Lx32** [comment]
Obv we cannot write Amazon Prime, but something like PrimeRomExtractor should be ok. Prime is no a trademarket word, is a common english word. But I'm not a lawyer

**Comment by RealRelativeEase** [comment]
Taking a page out of [plombo's](https://github.com/plombo/vcromclaim) book, how about NGPrimeClaim?

**Comment by zilladotexe** [comment]
It would be a huge shame if this project isn't completed for the final few games :(

**Comment by RedundantCich** [comment]
I think, Metal Slug 4 and the newer KoF are just too complex for our skills and knowledge :D 
Or do you have some other title in mind, @domholmes?

**Comment by zilladotexe** [comment]
> I think, Metal Slug 4 and the newer KoF are just too complex for our skills and knowledge :D Or do you have some other title in mind, @domholmes?

Ah sorry, they all look covered after your PR is merged, right? Thanks again for the work.

**Comment by lioneltrs** [comment]
Hi, I did not find a better name. I just spent some time checking the 23 scripts, and I corrected a few minors things. [RedundantCich](https://github.com/RedundantCich) did a great work this summer, expanding the scripts and the list of supported games. Thank you!

**Comment by RedundantCich** [comment]
Hey, Lionel! Awesome to see you back 😁
